packages/sql-json-schema-mapper/index.js

```diff
@@ -72,6 +73,9 @@ function mapSQLEntityToJSONSchema (entity) {
     if (field.isNullable) {
       properties[field.camelcase].nullable = true
     }
+    if (field.primaryKey) {
+      primaryKey.push(field.camelcase)
```

Comment:
Try attaching it to `properties[field.camelcase]`

Reply:
> Try attaching it to `properties[field.camelcase]`

Ok, I will do.

I had think to use a separated array (primaryKey) because the primary key could be the union of more fields.

Reply:
done... but tests fail.